pkg/vgmanager/vgmanager_controller.go

```diff
@@ -381,3 +382,14 @@ func (r *VGReconciler) getNewNodeStatus(status *lvmv1alpha1.VGStatus) *lvmv1alph
 	setStatus(status, vgNodeStatus)
 	return vgNodeStatus
 }
+
+func (r *VGReconciler) matchesMyNode(ctx context.Context, selector *corev1.NodeSelector) (bool, error) {
```

Comment:
need a better name than `matchesMynode`.

Reply:
Suggestions?

Reply:
`matchesNode`.   But I also agree with Leela's comment that we can remove this function.

Reply:
I disagree - functions provide readability.